Cannot install on Linux Fedora 40 from source

**Describe the bug:**

make install gives

go: downloading github.com/kishansagathiya/go-dot v0.1.0
graph.go:9:2: reading github.com/kishansagathiya/go-dot/go.mod at revision v0.1.0: git ls-remote -q origin in /root/go/pkg/mod/cache/vcs/0984fd59e6b4c7ab7bbd08ed009bd7fd280c3ab064deedd71f8f16de121de312: exit status 128:

        (gnome-ssh-askpass:49225): Gtk-WARNING **: 00:43:24.745: cannot open display:
        error: unable to read askpass response from '/usr/libexec/openssh/gnome-ssh-askpass'
        fatal: could not read Username for 'https://github.com': terminal prompts disabled
Confirm the import path was entered correctly.
If this is a private repository, see https://golang.org/doc/faq#git_https for additional information

Any idea? thanks

--- a/cmd/ipfs-cluster-ctl/graph.go
+++ b/cmd/ipfs-cluster-ctl/graph.go
@@ -6,7 +6,7 @@ import (
 	"io"
 	"sort"
 
-	dot "github.com/kishansagathiya/go-dot"
+	dot "github.com/ipfs-cluster/go-dot"
 	peer "github.com/libp2p/go-libp2p/core/peer"
 
 	"github.com/ipfs-cluster/ipfs-cluster/api"

--- a/go.mod
+++ b/go.mod
@@ -17,6 +17,7 @@ require (
 	github.com/hashicorp/raft v1.6.1
 	github.com/hashicorp/raft-boltdb v0.0.0-20190605210249-ef2e128ed477
 	github.com/hsanjuan/ipfs-lite v1.8.2
+	github.com/ipfs-cluster/go-dot v0.0.0-20240429211553-7274b631c71d
 	github.com/ipfs/boxo v0.19.0
 	github.com/ipfs/go-block-format v0.2.0
 	github.com/ipfs/go-cid v0.4.1
@@ -37,7 +38,6 @@ require (
 	github.com/ipld/go-codec-dagpb v1.6.0
 	github.com/ipld/go-ipld-prime v0.21.0
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/kishansagathiya/go-dot v0.1.0
 	github.com/lanzafame/go-libp2p-ocgorpc v0.1.1
 	github.com/libp2p/go-libp2p v0.33.2
 	github.com/libp2p/go-libp2p-consensus v0.0.1

